version_history.md

```diff
@@ -1,7 +1,9 @@
 ## ROCm Version History
 This file contains archived version history information for the [ROCm project](https://github.com/RadeonOpenCompute/ROCm)
 
-### Current ROCm Version: 2.7.2
+### Current ROCm Version: 2.8
+- [New features and enhancements in ROCm 2.7.2](#new-features-and-enhancements-in-rocm-272)
+- [New features and enhancements in ROCm 2.7.1](#new-features-and-enhancements-in-rocm-271)
```

Comment:
And drop the link for the 2.7.1 section.

Reply:
done